packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -38,15 +38,28 @@ export class JSONFormat extends React.Component<IJSONFormatProps, {}> {
 
     public render() {
         const { children, omitQuotesOnStrings, stringify } = this.props;
+
+        const isNully = children === undefined || children === null;
```

Comment:
`== null` is the only valid case of double-equals cuz it performs both of the checks you've done here